src/main/java/racingcar/model/Cars.java

```diff
@@ -0,0 +1,55 @@
+package racingcar.model;
+
+import racingcar.controller.CarStatusDto;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cars {
+
+    private static final int MIN_WINNER_POSITION = 0;
+    private final List<Car> cars;
+
+    public Cars(List<String> names) {
+        cars = initCars(names);
+    }
+
+    private List<Car> initCars(List<String> names) {
+        List<Car> resultCars = new ArrayList<>();
+        names.forEach(
+                name -> resultCars.add(new Car(name))
+        );
+        return resultCars;
+    }
```

Comment:
`List<Car>` 를 받는 생성자가 있다면 좀 더 유연한 테스트가 가능하지 않을까요? 

```suggestion
    public Cars(List<String> names) {
        cars = initCars(names);
    }

    private List<Car> initCars(List<String> names) {
        return names.stream()
            .map(it -> new Car(it))
            .collect(Collectors.toList());
    }
```

stream을 사용하는 것도 괜찮아 보이네요 :)